spec/factories/ratings.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :rating do
     user
-    application
+    association :rateable
```

Comment:
This is were the test break. I think it can be fixed with `association :rateable, factory: :application`, but then again, I don't know why it is a polymorphic association. We only rate applications, right?